AdaptiveUiCodelab/app/src/main/java/com/example/reply/data/local/LocalEmailsDataProvider.kt

```diff
@@ -116,7 +116,7 @@ object LocalEmailsDataProvider {
             """.trimIndent(),
             createAt = "20 mins ago",
             isStarred = true,
-            threads = threads,
+            replies = threads,
```

Comment:
Nit: if `replies` is changing names from `threads`, it would be good to update all usages of `threads`.

Reply:
Done